catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java

```diff
@@ -90,6 +90,23 @@ public String generateDropDatabaseSql(String databaseName, boolean cascade) {
     return dropDatabaseSql;
   }
 
+  @Override
+  protected void validateDatabaseName(String databaseName) {
+    if (StringUtils.isEmpty(databaseName)) {
+      throw new IllegalArgumentException("Database name cannot be empty.");
+    }
+    if (databaseName.length() > 64) {
+      throw new IllegalArgumentException("Database name cannot be longer than 64 characters.");
+    }
+    // Regex that matches any string that consists entirely of word characters, Unicode letters, or
+    // dollar signs
+    // \w matches [a-zA-Z0-9_]
+    // \p{L} matches any kind of letter from any language
+    if (!databaseName.matches("^[\\w\\p{L}$]*$")) {
+      throw new IllegalArgumentException("Invalid database name.");
```

Comment:
Please include the invalid database name in the error message to make it clearer for users.